src/index.js

```diff
@@ -0,0 +1,171 @@
+import './styles.css';
+import { ShowElement, HideElement } from './visiable.js';
+import CheckIfCompleted from './completion.js';
+
+let tasks = [];
+
+const enterIcon = document.getElementById('enter-icon');
+const addField = document.getElementById('add-field');
+const cleanBtn = document.getElementById('clean-btn');
+const listContainer = document.getElementById('list-container');
+let ShowList;
+
+if (localStorage.getItem('to_do_list') !== null) {
+  tasks = JSON.parse(localStorage.getItem('to_do_list'));
+}
+
+const UpdateStorage = () => localStorage.setItem('to_do_list', JSON.stringify(tasks));
+
+const UpdateIndex = (x) => {
+  for (let i = x; i < tasks.length; i += 1) {
+    tasks[i].index = i;
+  }
+  UpdateStorage();
+  ShowList();
+};
+
+const EditFun = (varX) => {
+  const editArrItem = tasks.filter((item) => item.index === varX);
```

Comment:
- kindly name your task id as `id` instead of index.
<img width="260" alt="image" src="https://user-images.githubusercontent.com/17850501/237037536-dd235813-f8b6-49a3-bcf9-ddb8dd7df8b2.png">
- as you can see from the image `id` would be more descriptive.